packages/tables/src/styled/StyledHeaderCell.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles } from '@zendeskgarden/react-theming';
+import { StyledCell } from './StyledCell';
+
+const COMPONENT_ID = 'tables.header_cell';
+
+export const StyledHeaderCell = styled(StyledCell.withComponent('th')).attrs({
```

Comment:
Prefer `as` attr vs the `withComponent` deprecation candidate.

Reply:
I've moved to the `as` prop in `attrs`.

Looking through the docs, [forwardedAs](https://www.styled-components.com/docs/api#forwardedas-prop) also looks really useful. We will have to watch our `peerDependency` range for some of these utilities as we are on the low-end for some of these props.